droidlet/perception/robot/handlers/detector.py

```diff
@@ -28,8 +29,10 @@
 
 lvis_yaml = "configs/mask_rcnn_R_101_FPN_1x.yaml"
 detector_weights = "model_999.pth"
-properties = "prop.pickle"
-things = "things.pickle"
+properties_default = "prop.pickle"
```

Comment:
as a follow-up to this PR, could you get rid of the defaults here? this will need updating the models tarball on s3, the protocol for which you can find here https://www.internalfb.com/intern/wiki/Droidlet/Protocol_for_uploading_models_datasets/